[2023-08-29 19:35:23,657] {processor.py:153} INFO - Started process (PID=18604) to work on /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:35:23,671] {processor.py:641} INFO - Processing file /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-29 19:35:23,674] {logging_mixin.py:115} INFO - [2023-08-29 19:35:23,674] {dagbag.py:507} INFO - Filling up the DagBag from /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:35:24,045] {logging_mixin.py:115} INFO - [2023-08-29 19:35:24,042] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-29 19:35:24,045] {logging_mixin.py:115} INFO - [2023-08-29 19:35:24,045] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-29 19:35:24,049] {processor.py:653} WARNING - No viable dags retrieved from /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:35:24,256] {processor.py:161} INFO - Processing /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.627 seconds
[2023-08-29 19:37:26,312] {processor.py:153} INFO - Started process (PID=19632) to work on /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:37:26,321] {processor.py:641} INFO - Processing file /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-08-29 19:37:26,324] {logging_mixin.py:115} INFO - [2023-08-29 19:37:26,323] {dagbag.py:507} INFO - Filling up the DagBag from /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:37:26,611] {logging_mixin.py:115} INFO - [2023-08-29 19:37:26,608] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-08-29 19:37:26,611] {logging_mixin.py:115} INFO - [2023-08-29 19:37:26,611] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-08-29 19:37:26,612] {processor.py:653} WARNING - No viable dags retrieved from /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-08-29 19:37:26,821] {processor.py:161} INFO - Processing /home/felipe/extracao_dados/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.594 seconds
